README.md

```diff
@@ -31,6 +31,66 @@ And below are the lower-level packages, used internally, or by advanced users.
 | [`@parity/contracts`](/packages/contracts) | [![npm (scoped)](https://img.shields.io/npm/v/@parity/contracts.svg)](https://www.npmjs.com/package/@parity/contracts) | [![README](https://img.shields.io/badge/docs-README-green.svg)](/packages/contracts#readme)      | Parity's [contracts](https://github.com/parity-contracts) as ES6 classes. |
 | [`@parity/electron`](/packages/electron)   | [![npm (scoped)](https://img.shields.io/npm/v/@parity/electron.svg)](https://www.npmjs.com/package/@parity/electron)   | [![README](https://img.shields.io/badge/docs-README-green.svg)](/packages/electron#readme)       | Control the Parity Ethereum node from Electron.                           |
 
+### Contributing
+
+#### Dependencies
+
+Install at least `yarn` version 1.4.2 and [Node.js >=10.11.0](https://nodejs.org/en/)
+
+```
+yarn --version // Should be at least 1.4.2
+```
+
+#### Tests
+
+```
+export NPM_TOKEN=''; yarn test
```

Comment:
No need for `export NPM_TOKEN='';`. Can you confirm @ltfschoen?

Reply:
Yes confirmed. It is no longer required

Reply:
I just created another issue. It looks like this issue came back again